src/lib/middleware/__tests__/redisPageCache.jest.ts

```diff
@@ -72,6 +78,10 @@ describe("pageCacheMiddleware", () => {
       method: "GET",
       path: "/artist-series/test-artist",
       url: "https://artsy.net/artist-series/test-artist",
+      // TODO: remove once "artist_grid_manual_curation_trial" A/B test is over
+      _parsedUrl: {
+        pathname: "/artist-series/test-artist",
+      },
```

Comment:
**question**: Is the artist-series page affected by this test (and cache) as well?

Reply:
Yep (If I understood your question correctly)

FYI These changes are necessary here in order to avoid such an error for [these lines](https://github.com/artsy/force/blob/2a803c67a5cd9547a3a07c0c548d3ab9755c90b7/src/lib/middleware/redisPageCache.ts#L61)(cannot get `pathname` from `req._parsedUrl`)
<img width="947" alt="image" src="https://user-images.githubusercontent.com/3513494/158143048-354898c9-d99d-43d3-8d6a-6aadc022e3ed.png">